tensorflow/compiler/xla/service/cpu/build_defs.bzl

```diff
@@ -10,3 +10,14 @@ def runtime_copts():
       "//tensorflow:android": ["-O2"],
       "//conditions:default": []
   }))
+
+def if_polly(basic_deps):
```

Comment:
This should not be called `if_polly`.  Given what it does today, it should be called `append_polly_dep_if_enabled`, though I think it is better to have a `polly_component_if_available` function that returns `["@llvm//:polly"]` if polly is enabled else returns an empty list; and explicitly appending `polly_component_if_available()` in the `deps` for various rules.

Reply:
Polly would be added as a dependency in just one component. Do you still suggest we take the second approach?